data/json/recipes/tools/tools_electronic.json

```diff
@@ -691,5 +691,36 @@
     "proficiencies": [ { "proficiency": "prof_plasticworking", "time_multiplier": 1.5 }, { "proficiency": "prof_elec_soldering" } ],
     "tools": [ [ [ "mold_plastic", -1 ] ] ],
     "components": [ [ [ "razor_blade", 1 ] ], [ [ "motor_micro", 1 ] ], [ [ "power_supply", 1 ] ], [ [ "plastic_chunk", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "radio_car",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_TOOLS",
+    "skill_used": "electronics",
+    "skills_required": [ [ "mechanics", 4 ], [ "fabrication", 6 ] ],
+    "difficulty": 6,
+    "time": "4 h 45 m",
+    "reversible": true,
+    "decomp_learn": 5,
+    "book_learn": [ [ "textbook_electronics", 7 ], [ "textbook_robots", 6 ] ],
+    "using": [ [ "soldering_standard", 10 ], [ "surface_heat", 10 ] ],
+    "qualities": [ { "id": "SCREW", "level": 1 } ],
+    "proficiencies": [ { "proficiency": "prof_plasticworking", "time_multiplier": 1.5 }, { "proficiency": "prof_elec_soldering" } ],
+    "tools": [ [ [ "mold_plastic", -1 ] ] ],
```

Comment:
Is this really the only tool?

Reply:
Plastic mold, screw driver, soldering iron, and a heat surface. What else should be added, a cutting tool?

Reply:
Maybe a metal saw to cut the large spring in to two smaller pieces for each wheel's suspension.